_rules/css-orientation-lock-media-queries-b33eff.md

```diff
@@ -0,0 +1,175 @@
+---
+id: b33eff
+name: Page has no orientation lock specified using CSS media queries
+rule_type: atomic
+description: |
+  This rule checks that page content is not locked to any specific display orientation using CSS media queries, and the content is operable in all display orientations.
+accessibility_requirements:
+ wcag20:1.3.4: # Orientation
+  forConformance: true
+  failed: not satisfied
+  passed: further testing needed
+  inapplicable: further testing needed
+input_aspects:
+- DOM Tree
+- CSS Styling
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to any `element` that is [visible](#visible) and has a CSS `transform` style with a [rotate](https://drafts.csswg.org/css-transforms/#funcdef-transform-rotate) filter, set from a CSS media query with an [orientation](https://drafts.csswg.org/mediaqueries-3/#orientation) media feature (`horizontal` or `vertical`).
```

Comment:
`transform` should be introduced as a property, not a style
`rotate` should be introduced as a transform-function, not a filter
`orientation` values are `landscape` and `portrait`, not `horizontal` and `vertical`